_docs/release-notes.md

```diff
@@ -351,6 +360,26 @@ Visit [https://tidepool.org/uploader](https://tidepool.org/uploader) to download
 
 If you already have the Tidepool Uploader installed on your computer, it will automatically update to the latest version.  
 
+### 2.6.5 (Released 2018-07-24)
+It's another Bug Fix Bonanza™! This this round is devoted to supported Medtronic 5/7 pumps.  
+
+We fixed a bug for pumpers who use dual or square boluses without extending their bolus (100%/0%).  
+[Feature Requirements](https://trello.com/c/3zeww3MJ/14-error-details-expected-a-normal-bolus-but-found-type-4-or-5)
+
+We fixed a bug that inelegantly handled how we display scheduled basal rates after a 24-hour temp basals. Things are much more elegant now.  
```

Comment:
Haha, I don't know if it's that elegant. Previously, the pump wouldn't upload in these cases. After the bug fix we now show a gap since we don't know the basal duration, as the pump does not provide us with this information. This is due to a bug in their firmware, but we don't have to call that out in the release notes.

Reply:
Particularly if this is impacted by a bug in their firmware, I'd consider this an elegant solution.